app/utils/mail/mailworker.py

```diff
@@ -1,13 +1,22 @@
 import smtplib
 import ssl
 from email.message import EmailMessage
+from pathlib import Path
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
     from app.core.config import Settings
 
 
-def send_email(recipient: str, subject: str, content: str, settings: "Settings"):
+def send_email(
+    recipient: str | list[str],
```

Comment:
I think we could ask for a `list[str]` directly instead of allowing `str | list[str]`

Reply:
This is a convenient system which allow to not have inconsistency such as using a list of email when trying to send a personal token, moreover I didn't want to touch the old system